share/spice/coderwall/coderwall.js

```diff
@@ -58,8 +52,35 @@
             },
 
             templates: {
-                group: 'icon'
+                group: 'icon',
+                variants: {
+                    iconImage: 'large'
+                },
+                options: {
+                    chompContents: true,
+                    content: DDH.coderwall.content
+                }
             }
         });
+
     };
+
+    Handlebars.registerHelper("parseMD", function(content){
```

Comment:
Did you get this from anywhere? If so it'd be useful to include a link to reference so peeps can check out more info.